src/fsharp/fsc.fsi

```diff
@@ -32,6 +32,9 @@ type ILResource with
 /// Proccess the given set of command line arguments
 val internal ProcessCommandLineFlags : TcConfigBuilder * setProcessThreadLocals:(TcConfigBuilder -> unit) * lcidFromCodePage : int option * argv:string[] -> string list
 
+module internal StaticLinker =
+    val internal StaticLink : TcConfig * TcImports * ILGlobals -> (ILTypeRef * ILTypeDef) list * (ILModuleDef -> ILModuleDef)
```

Comment:
Where is this called?  Ah, in fsc.fs, but the diff is not showing in github.  Any chance you can make the diff there smaller, e.g. by adjusting indentation levels?